audit.md

```diff
@@ -0,0 +1,114 @@
+
+## JS
+  1. Setup JavaScript/Solidity linters, such as eslint, advantages:
+    1. keeping your code style consistent
+    2. Spotting potential errors and bad patterns (duplicate variables, unreachable code or invalid regular expressions)
+    3. Saving time (you might uncover a bug or two on early stages, before they even reach runtime)
+on a side note: use prettier, it is a tool that will help you with formatting your code. You can run prettier from command line or use it at pre commit hook
+  
+  2. Add truffle as dependency and write scripts for it, so the developers won&#39;t need to install it globally
+  
+  3. Use ES Next features such as
+- let and const instead of var for variable declarations
+- function default parameters
+- template literals
+- async/await
+- in most places arrow you can use arrow functions instead of function declaration
+
+ 4. Write DRY code:
+- There are a lot of places where code is duplicated (we found 10^18 to be used ~20 times)
+- functions ether and coin are identical
+- come up with better variable names, b1 and b2 does say nothing to developers
+
+
+  5. Script tags should be placed in head instead at the bottom of the body (add defer attribute to the tag)
+
+  6. Missing properties in package.json: name and version.  A **package.json** must have:
+
+- **&quot;name&quot;**
+  - all lowercase
+  - one word, no spaces
+  - dashes and underscores allowed
+- **&quot;version&quot;**
+  - in the form of **x.x.x**
+  - follows semver spec. It's also good to have such fields as: author, license, contributors, description  and scripts. A **package.json** file:
+
+- lists the packages that your project depends on.
+- allows you to specify the versions of a package that your project can use using semantic versioning rules.
+- makes your build reproducible, and therefore _much_ easier to share with other developers.
+
+## Suggestions about contracts code:
+
+1. Logic is very hardly visible in code. Only single contract inheritance is used which causes that main contract FollowCoin.sol has only 1 public function that doesn&#39;t tell much about logic. My suggestion is to use multiple inheritance where possible to avoid 6 elements long inheritance chain.
+
+2. Tests seem to be only negative (are testing when functions should fail) but they should also be positive (test when function should pass)
+
+3. Migration passed 5 arguments to contract while it expected for 0 arguments: should be fixed in this PR: [https://github.com/FollowCoin/followcoin-contract/pull/4](https://github.com/FollowCoin/followcoin-contract/pull/4)
+
+4. Smart Contract code often is not DRY (Do not Repeat Yourself rule), for example in <strong>transferFrom</strong> function which is overloaded couple of times, but instead of having common base for those functions, each overload copies the same code and changes just one line.
+
+## Questions for feature verification
+
+##### Can you transfer the FLLWB from person A to person B?
+Yes
+
```

Comment:
I would also suggest to answer this question with some argumentation, not just Yes / No.